genie-core/src/main/java/com/netflix/genie/core/events/JobScheduledEvent.java

```diff
@@ -32,25 +36,37 @@
 @Getter
 public class JobScheduledEvent extends BaseJobEvent {
```

Comment:
So we talked about renaming this cause not it's more of a "schedule a job event" and that's something that might be better as a direct method call to a service than an event? The events are intended to be listened and acted on by multiple listeners that are idempotent. This seems more like a direct action that needs to be taken.